data_base/squema.sql

```diff
@@ -38,4 +38,23 @@ CREATE TABLE album (
   item_id  INT REFERENCES item(id),
   on_spotify  BOOLEAN,
   PRIMARY KEY(item_id)
+);
+
+CREATE TABLE label(
+  id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+  title VARCHAR(32),
+  color VARCHAR(20),
+);
+
+CREATE TABLE book(
+  id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+  genre_id INT,
+  author_id INT,
+  label_id INT,
+  publish_date DATE,
```

Comment:
Kindly suggest you use a foreign key to use the item as a father in order to keep your code DRY